modules/scheduler/gke-cluster/variables.tf

```diff
@@ -170,11 +170,7 @@ variable "system_node_pool_taints" {
     value  = any
     effect = string
   }))
-  default = [{
```

Comment:
are we removing the system np taint as well? I thought we were only discussing removing the manual gpu taint

Reply:
Sounds good, keeping this for now.